### ISSUE TITLE
Error with android:exported on installation

### ISSUE THREAD

**Issue by Ashkan4472** [pusheco/pushe-flutter#30]
When user use pushe-flutter package, there are no error on build but for installation there is this error:
```
failed to install build/app/outputs/flutter-apk/app.apk: Failure [INSTALL_PARSE_FAILED_MANIFEST_MALFORMED: Failed parse during installPackageLI:
/data/app/vmdl1104452551.tmp/base.apk (at Binary XML file line #155): co.pushe.plus.fcm.FcmService: Targeting S+ (version 31 and above) requires that an explicit value for android:exported be
defined when intent filters are present]
Error running application on SM G998B.
```
apparently there has to be ```android:exported="true|false"```` on any activity you use if targetBuildSDK is 31 or above.

please fix this issue

**Comment by mahdi-malv** [comment]
Hi

Thanks for reaching out. The update will be published as soon as the native dependecy (2.5.1-rc02) gets stable (which will happen soon).

Issue is actually from the native library ([Link to](https://github.com/pusheco/pushe-android-studio-sample/issues/49))

New version will be published thereafter.

**Comment by mahdi-malv** [comment]
Fixed in `2.5.1`
